cinch/roles/jenkins_slave/defaults/main.yml

```diff
@@ -7,6 +7,8 @@ jenkins_url: http://example.com:8080/jenkins/
 jslave_name: "{{ inventory_hostname }}"
 # The label for this slave
 jslave_label: ops-jslave
+# Additional list of variables injected into jswarm's environment
+jswarm_environment: []
```

Comment:
Can you give a little documentation here about the format these variables are supposed to be expressed in?